ui/src/app/views/signed-in/component.html

```diff
@@ -62,7 +62,7 @@ <h6 style="padding: 0.5rem;">
       <button class="btn btn-primary nav-link"
           [class.active]="createWorkspaceActive"
           routerLink="/workspaces/build"
-          [disabled]="(hasDataAccess ? null : true) || !billingProjectInitialized"
```

Comment:
Should remove parens here for consistency with the other button.  OK to merge without this.